File: backend/src/modules/avaliacao_fisica/entity/AvaliacaoFisica.js
class AvaliacaoFisica {
  constructor({ id_avaliacao, data_avaliacao, peso, altura, observacoes, aluno_id_aluno }) {
    this.id_avaliacao = id_avaliacao;
    this.data_avaliacao = data_avaliacao;
    this.peso = peso;
    this.altura = altura;
    this.observacoes = observacoes;
    this.aluno_id_aluno = aluno_id_aluno;
  }
  validate() {
    const errors = [];
    if (!this.aluno_id_aluno) errors.push('Aluno é obrigatório');
    return errors;
  }
}
module.exports = AvaliacaoFisica;

File: backend/src/main/java/com/vitalativa/entity/AvaliacaoFisica.java
package com.vitalativa.entity;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import java.time.LocalDate;

@Entity
@Table(name = "avaliacao_fisica")
public class AvaliacaoFisica {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_avaliacao")
    private Integer idAvaliacao;
    
    @Column(name = "data_avaliacao")
    private LocalDate dataAvaliacao;
    
    private Double peso;
    private Double altura;
    private String observacoes;
    
    @Column(name = "aluno_id_aluno")
    private Integer alunoIdAluno;
    
    public AvaliacaoFisica() {}
    
    public Integer getIdAvaliacao() { return idAvaliacao; }
    public void setIdAvaliacao(Integer idAvaliacao) { this.idAvaliacao = idAvaliacao; }
    public LocalDate getDataAvaliacao() { return dataAvaliacao; }
    public void setDataAvaliacao(LocalDate dataAvaliacao) { this.dataAvaliacao = dataAvaliacao; }
    public Double getPeso() { return peso; }
    public void setPeso(Double peso) { this.peso = peso; }
    public Double getAltura() { return altura; }
    public void setAltura(Double altura) { this.altura = altura; }
    public String getObservacoes() { return observacoes; }
    public void setObservacoes(String observacoes) { this.observacoes = observacoes; }
    public Integer getAlunoIdAluno() { return alunoIdAluno; }
    public void setAlunoIdAluno(Integer alunoIdAluno) { this.alunoIdAluno = alunoIdAluno; }
}